projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/style.scss

```
@@ -0,0 +1,51 @@
+.value-section {
+    display: flex;
+    justify-content: space-between;
+
+    &__heading {
+        font-size: 12px;
+        color: var(--jp-gray-100);
+        font-weight: 500;
+    }
+
+    &__last-scan {
+        display: flex;
+        align-items: center;
+        margin-top: var(--spacing-base);
+        column-gap: 1px;
+        position: absolute;
+        right: calc(var(--spacing-base)* 3);
```

Nitpick, weird looking spacing here
```suggestion
        right: calc(var(--spacing-base)*3);
```